primitives/avn-common/src/tests/helpers.rs

```diff
@@ -10,6 +10,7 @@ pub mod ethereum_converters {
         return vec
     }
 
+    #[cfg(test)]
```

Comment:
This is a minor thing, but do we need this here? If this is a test helper, maybe we should only qualify with this directive the inclusion of the file itself in the callers? We wouldn't take part of the test helper without taking the rest, would we?